pkg/addon/policyframework/manifests/managedclusterchart/templates/namespace.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright Contributors to the Open Cluster Management project
+
+{{- if not .Values.onMulticlusterHub }}
+apiVersion: v1
+kind: Namespace
```

Comment:
Are you also going to remove the code to create/delete cluster ns in spec sync and status sync?

Reply:
Yes, after this is finished and merged, the creation in spec sync can be removed, then the permissions it used. Doing them one PR at a time will make sure builds don't break if an image push or something fails (since that has been acting up lately).

Deletion in status sync can be removed independently, with no concerns for breaking anything - it's already not used in 2.5.